ipv8/attestation/communication_manager.py

```diff
@@ -29,18 +34,19 @@ def __init__(self, attestation_community, identity_community):
         self.attestation_overlay.set_verify_request_callback(self.on_verify_request)
 
     @property
-    def public_key_bin(self):
+    def public_key_bin(self) -> bytes:
         return self.identity_overlay.my_peer.public_key.key_to_bin()
 
     @property
-    def peers(self):
+    def peers(self) -> typing.List[Peer]:
```

Comment:
Why not use `List` here? I think that is more explicit and clear. The fact that it's in the `typing` library should not be relevant here.

Reply:
This is faster* and `typing` was already imported anyway. Here's the proof:

```python
print(timeit.timeit("import typing; a = typing.List[str]", number=1000000))
> 2.016579663999437
print(timeit.timeit("import typing; from typing import List; a = List[str]", number=1000000))
> 4.145822678998229
```

Is `List` more readable than `typing.List`? Yes**. However, I don't believe it's such a dramatic improvement that it warrants the 2x slow-down though.

*Because it happens at module-load time and not at running time (e.g. in a loop).
**Sort of, `typing.List` leaves less room for ambiguity.

---

All of that said, this is a discussion about aesthetics, personal taste and microsecond optimizations. I implore you not to start this discussion here and let this one slide 😃